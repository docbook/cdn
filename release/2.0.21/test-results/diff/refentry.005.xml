<html xmlns:deltaxml="http://www.deltaxml.com/ns/well-formed-delta-v1" xmlns="http://www.w3.org/1999/xhtml" xmlns:dxx="http://www.deltaxml.com/ns/xml-namespaced-attribute" xmlns:dxa="http://www.deltaxml.com/ns/non-namespaced-attribute" deltaxml:deltaV2="A!=B" deltaxml:version="2.0" deltaxml:content-type="full-context">
   <head deltaxml:deltaV2="A=B">
      <link href="../../resources/css/db-prism.css" rel="stylesheet" type="text/css"/>
      <link rel="stylesheet" type="text/css" href="../../resources/css/default.css"/>
      <link href="../../resources/css/prism.css" rel="stylesheet" type="text/css"/>
      <meta charset="utf-8"/>
      <meta name="generator" content="DocBook XSL 2.0 Stylesheets Vxxx"/>
      <script type="text/javascript" src="../../resources/js/dbmodnizr.js"/>
      <title>Also Minimal Title</title>
   </head>
   <body deltaxml:deltaV2="A!=B">
      <article deltaxml:deltaV2="A!=B" id="lessminimal" class="refentry">
         <div deltaxml:deltaV2="A" class="refnamediv">
            <h2>Name</h2>
            <p>
               <span class="refname">Also Minimal</span>
               <span class="refpurpose">
                  <span class="refpurpose-sep"> — </span>
                  <span class="refpurpose-text">Another test of a minimal RefEntry</span>
                  <span class="refpurpose-punc">.</span>
               </span>
            </p>
         </div><deltaxml:textGroup deltaxml:deltaV2="A"><deltaxml:text deltaxml:deltaV2="A">
         </deltaxml:text></deltaxml:textGroup><div deltaxml:deltaV2="A!=B"><deltaxml:attributes deltaxml:deltaV2="A!=B"><dxa:class deltaxml:deltaV2="A!=B"><deltaxml:attributeValue deltaxml:deltaV2="A">refsection</deltaxml:attributeValue><deltaxml:attributeValue deltaxml:deltaV2="B">content</deltaxml:attributeValue></dxa:class></deltaxml:attributes>
            <div deltaxml:deltaV2="A!=B"><deltaxml:attributes deltaxml:deltaV2="A!=B"><dxa:class deltaxml:deltaV2="A!=B"><deltaxml:attributeValue deltaxml:deltaV2="A">refsection-titlepage</deltaxml:attributeValue><deltaxml:attributeValue deltaxml:deltaV2="B">refnamediv</deltaxml:attributeValue></dxa:class></deltaxml:attributes>
               <h2 deltaxml:deltaV2="A!=B"><deltaxml:textGroup deltaxml:deltaV2="A!=B"><deltaxml:text deltaxml:deltaV2="A">Description</deltaxml:text><deltaxml:text deltaxml:deltaV2="B">Name</deltaxml:text></deltaxml:textGroup></h2><deltaxml:textGroup deltaxml:deltaV2="B"><deltaxml:text deltaxml:deltaV2="B">
               </deltaxml:text></deltaxml:textGroup><p deltaxml:deltaV2="B">
                  <span class="refname">Also Minimal</span>
                  <span class="refpurpose">
                     <span class="refpurpose-sep"> — </span>
                     <span class="refpurpose-text">Another test of a minimal RefEntry</span>
                     <span class="refpurpose-punc">.</span>
                  </span>
               </p>
            </div>
            <p deltaxml:deltaV2="A">This is a minimal RefEntry.</p><div deltaxml:deltaV2="B" class="refsection">
               <div class="refsection-titlepage">
                  <h2>Description</h2>
               </div>
               <div class="content">
                  <p>This is a minimal RefEntry.</p>
                  <div class="refsection">
                     <div class="refsection-titlepage">
                        <h3>Subsection</h3>
                     </div>
                     <div class="content">
                        <p>This is a minimal RefEntry.</p>
                        <div class="refsection">
                           <div class="refsection-titlepage">
                              <h4>Sub-subsection</h4>
                           </div>
                           <div class="content">
                              <p>This is a minimal RefEntry.</p>
                           </div>
                        </div>
                     </div>
                  </div>
               </div>
            </div>
            <div deltaxml:deltaV2="A!=B" class="refsection">
               <div deltaxml:deltaV2="A!=B" class="refsection-titlepage">
                  <h3 deltaxml:deltaV2="A">Subsection</h3><h2 deltaxml:deltaV2="B">More Description</h2>
               </div>
               <p deltaxml:deltaV2="A">This is a minimal RefEntry.</p><deltaxml:textGroup deltaxml:deltaV2="A"><deltaxml:text deltaxml:deltaV2="A">
               </deltaxml:text></deltaxml:textGroup><div deltaxml:deltaV2="A!=B"><deltaxml:attributes deltaxml:deltaV2="A!=B"><dxa:class deltaxml:deltaV2="A!=B"><deltaxml:attributeValue deltaxml:deltaV2="A">refsection</deltaxml:attributeValue><deltaxml:attributeValue deltaxml:deltaV2="B">content</deltaxml:attributeValue></dxa:class></deltaxml:attributes>
                  <div deltaxml:deltaV2="A" class="refsection-titlepage">
                     <h4>Sub-subsection</h4>
                  </div><deltaxml:textGroup deltaxml:deltaV2="A"><deltaxml:text deltaxml:deltaV2="A">
                  </deltaxml:text></deltaxml:textGroup><p deltaxml:deltaV2="A!=B"><deltaxml:textGroup deltaxml:deltaV2="A!=B"><deltaxml:text deltaxml:deltaV2="A">This is a minimal RefEntry.</deltaxml:text><deltaxml:text deltaxml:deltaV2="B">This is a not-so minimal RefEntry.</deltaxml:text></deltaxml:textGroup></p>
               </div>
            </div><deltaxml:textGroup deltaxml:deltaV2="B"><deltaxml:text deltaxml:deltaV2="B">
            </deltaxml:text></deltaxml:textGroup><div deltaxml:deltaV2="B" class="refsection">
               <div class="refsection-titlepage">
                  <h2>Even More Description</h2>
               </div>
               <div class="content">
                  <p>This is an even less minimal RefEntry.</p>
               </div>
            </div>
         </div><deltaxml:textGroup deltaxml:deltaV2="A"><deltaxml:text deltaxml:deltaV2="A">
         </deltaxml:text></deltaxml:textGroup><div deltaxml:deltaV2="A" class="refsection">
            <div class="refsection-titlepage">
               <h2>More Description</h2>
            </div>
            <p>This is a not-so minimal RefEntry.</p>
         </div><deltaxml:textGroup deltaxml:deltaV2="A"><deltaxml:text deltaxml:deltaV2="A">
         </deltaxml:text></deltaxml:textGroup><div deltaxml:deltaV2="A" class="refsection">
            <div class="refsection-titlepage">
               <h2>Even More Description</h2>
            </div>
            <p>This is an even less minimal RefEntry.</p>
         </div>
      </article>
      <script deltaxml:deltaV2="A=B" src="../../resources/js/prism.js"/>
   </body>
</html>