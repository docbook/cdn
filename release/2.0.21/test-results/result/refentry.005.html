<html xmlns="http://www.w3.org/1999/xhtml" xmlns:p="http://www.w3.org/ns/xproc" p:style=""><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8" /><title>Results for refentry.005.xml</title><link rel="stylesheet" type="text/css" href="../../resources/css/default.css" /><script type="text/javascript" src="../../resources/js/dbmodnizr.js"></script><link rel="stylesheet" type="text/css" href="http://ajax.googleapis.com/ajax/libs/jqueryui/1.11.4/themes/start/jquery-ui.css" /><link rel="stylesheet" type="text/css" href="../../resources/css/prism.css" /><link rel="stylesheet" type="text/css" href="../../resources/css/db-prism.css" /><link href="../style/show-results.css" rel="stylesheet" type="text/css" /><meta name="deltaxml" content="true" /></head><body><h1>DocBook XSLT 2.0 Stylesheet output: refentry.005.xml</h1><table class="diffs"><tbody><tr><th>Expected (<span class="exdiff">17</span> differences)</th><th>Actual (<span class="adiff">11</span> differences)</th></tr><tr><td><pre class="show-diff"><span class="tag diff">&lt;html&gt;
   <span class="tag nodiff">&lt;head&gt;
      <span class="tag">&lt;link <span class="att">href="../../resources/css/db-prism.css"</span> <span class="att">rel="stylesheet"</span> <span class="att">type="text/css"</span>/&gt;</span>
      <span class="tag">&lt;link <span class="att">href="../../resources/css/default.css"</span> <span class="att">rel="stylesheet"</span> <span class="att">type="text/css"</span>/&gt;</span>
      <span class="tag">&lt;link <span class="att">href="../../resources/css/prism.css"</span> <span class="att">rel="stylesheet"</span> <span class="att">type="text/css"</span>/&gt;</span>
      <span class="tag">&lt;meta <span class="att">charset="utf-8"</span>/&gt;</span>
      <span class="tag">&lt;meta <span class="att">content="DocBook XSL 2.0 Stylesheets Vxxx"</span> <span class="att">name="generator"</span>/&gt;</span>
      <span class="tag">&lt;script <span class="att">src="../../resources/js/dbmodnizr.js"</span> <span class="att">type="text/javascript"</span>/&gt;</span>
      <span class="tag">&lt;title&gt;Also Minimal Title&lt;/title&gt;</span>
   &lt;/head&gt;</span>
   <span class="tag diff">&lt;body&gt;
      <span class="tag diff">&lt;article <span class="att">class="refentry"</span> <span class="att">id="lessminimal"</span>&gt;
         <span class="tag diff del">&lt;div <span class="att">class="refnamediv"</span>&gt;
            <span class="tag">&lt;h2&gt;Name&lt;/h2&gt;</span>
            <span class="tag">&lt;p&gt;
               <span class="tag">&lt;span <span class="att">class="refname"</span>&gt;Also Minimal&lt;/span&gt;</span>
               <span class="tag">&lt;span <span class="att">class="refpurpose"</span>&gt;
                  <span class="tag">&lt;span <span class="att">class="refpurpose-sep"</span>&gt; — &lt;/span&gt;</span>
                  <span class="tag">&lt;span <span class="att">class="refpurpose-text"</span>&gt;Another test of a minimal RefEntry&lt;/span&gt;</span>
                  <span class="tag">&lt;span <span class="att">class="refpurpose-punc"</span>&gt;.&lt;/span&gt;</span>
               &lt;/span&gt;</span>
            &lt;/p&gt;</span>
         &lt;/div&gt;</span><span class="diff txtchg">
         </span><span class="tag diff">&lt;div <span class="att">class="<span class="att attchg">refsection</span>"</span>&gt;
            <span class="tag diff">&lt;div <span class="att">class="<span class="att attchg">refsection-titlepage</span>"</span>&gt;
               <span class="tag diff">&lt;h2&gt;<span class="diff txtchg">Description</span>&lt;/h2&gt;</span>
            &lt;/div&gt;</span>
            <span class="tag diff del">&lt;p&gt;This is a minimal RefEntry.&lt;/p&gt;</span>
            <span class="tag diff">&lt;div <span class="att">class="refsection"</span>&gt;
               <span class="tag diff">&lt;div <span class="att">class="refsection-titlepage"</span>&gt;
                  <span class="tag diff del">&lt;h3&gt;Subsection&lt;/h3&gt;</span>
               &lt;/div&gt;</span>
               <span class="tag diff del">&lt;p&gt;This is a minimal RefEntry.&lt;/p&gt;</span><span class="diff txtchg">
               </span><span class="tag diff">&lt;div <span class="att">class="<span class="att attchg">refsection</span>"</span>&gt;
                  <span class="tag diff del">&lt;div <span class="att">class="refsection-titlepage"</span>&gt;
                     <span class="tag">&lt;h4&gt;Sub-subsection&lt;/h4&gt;</span>
                  &lt;/div&gt;</span><span class="diff txtchg">
                  </span><span class="tag diff">&lt;p&gt;<span class="diff txtchg">This is a minimal RefEntry.</span>&lt;/p&gt;</span>
               &lt;/div&gt;</span>
            &lt;/div&gt;</span>
         &lt;/div&gt;</span><span class="diff txtchg">
         </span><span class="tag diff del">&lt;div <span class="att">class="refsection"</span>&gt;
            <span class="tag">&lt;div <span class="att">class="refsection-titlepage"</span>&gt;
               <span class="tag">&lt;h2&gt;More Description&lt;/h2&gt;</span>
            &lt;/div&gt;</span>
            <span class="tag">&lt;p&gt;This is a not-so minimal RefEntry.&lt;/p&gt;</span>
         &lt;/div&gt;</span><span class="diff txtchg">
         </span><span class="tag diff del">&lt;div <span class="att">class="refsection"</span>&gt;
            <span class="tag">&lt;div <span class="att">class="refsection-titlepage"</span>&gt;
               <span class="tag">&lt;h2&gt;Even More Description&lt;/h2&gt;</span>
            &lt;/div&gt;</span>
            <span class="tag">&lt;p&gt;This is an even less minimal RefEntry.&lt;/p&gt;</span>
         &lt;/div&gt;</span>
      &lt;/article&gt;</span>
      <span class="tag nodiff">&lt;script <span class="att">src="../../resources/js/prism.js"</span>/&gt;</span>
   &lt;/body&gt;</span>
&lt;/html&gt;</span></pre></td><td><pre class="show-diff"><span class="tag diff">&lt;html&gt;
   <span class="tag nodiff">&lt;head&gt;
      <span class="tag">&lt;link <span class="att">href="../../resources/css/db-prism.css"</span> <span class="att">rel="stylesheet"</span> <span class="att">type="text/css"</span>/&gt;</span>
      <span class="tag">&lt;link <span class="att">href="../../resources/css/default.css"</span> <span class="att">rel="stylesheet"</span> <span class="att">type="text/css"</span>/&gt;</span>
      <span class="tag">&lt;link <span class="att">href="../../resources/css/prism.css"</span> <span class="att">rel="stylesheet"</span> <span class="att">type="text/css"</span>/&gt;</span>
      <span class="tag">&lt;meta <span class="att">charset="utf-8"</span>/&gt;</span>
      <span class="tag">&lt;meta <span class="att">content="DocBook XSL 2.0 Stylesheets Vxxx"</span> <span class="att">name="generator"</span>/&gt;</span>
      <span class="tag">&lt;script <span class="att">src="../../resources/js/dbmodnizr.js"</span> <span class="att">type="text/javascript"</span>/&gt;</span>
      <span class="tag">&lt;title&gt;Also Minimal Title&lt;/title&gt;</span>
   &lt;/head&gt;</span>
   <span class="tag diff">&lt;body&gt;
      <span class="tag diff">&lt;article <span class="att">class="refentry"</span> <span class="att">id="lessminimal"</span>&gt;
         <span class="tag diff">&lt;div <span class="att">class="<span class="att attchg">content</span>"</span>&gt;
            <span class="tag diff">&lt;div <span class="att">class="<span class="att attchg">refnamediv</span>"</span>&gt;
               <span class="tag diff">&lt;h2&gt;<span class="diff txtchg">Name</span>&lt;/h2&gt;</span><span class="diff txtchg">
               </span><span class="tag diff add">&lt;p&gt;
                  <span class="tag">&lt;span <span class="att">class="refname"</span>&gt;Also Minimal&lt;/span&gt;</span>
                  <span class="tag">&lt;span <span class="att">class="refpurpose"</span>&gt;
                     <span class="tag">&lt;span <span class="att">class="refpurpose-sep"</span>&gt; — &lt;/span&gt;</span>
                     <span class="tag">&lt;span <span class="att">class="refpurpose-text"</span>&gt;Another test of a minimal RefEntry&lt;/span&gt;</span>
                     <span class="tag">&lt;span <span class="att">class="refpurpose-punc"</span>&gt;.&lt;/span&gt;</span>
                  &lt;/span&gt;</span>
               &lt;/p&gt;</span>
            &lt;/div&gt;</span>
            <span class="tag diff add">&lt;div <span class="att">class="refsection"</span>&gt;
               <span class="tag">&lt;div <span class="att">class="refsection-titlepage"</span>&gt;
                  <span class="tag">&lt;h2&gt;Description&lt;/h2&gt;</span>
               &lt;/div&gt;</span>
               <span class="tag">&lt;div <span class="att">class="content"</span>&gt;
                  <span class="tag">&lt;p&gt;This is a minimal RefEntry.&lt;/p&gt;</span>
                  <span class="tag">&lt;div <span class="att">class="refsection"</span>&gt;
                     <span class="tag">&lt;div <span class="att">class="refsection-titlepage"</span>&gt;
                        <span class="tag">&lt;h3&gt;Subsection&lt;/h3&gt;</span>
                     &lt;/div&gt;</span>
                     <span class="tag">&lt;div <span class="att">class="content"</span>&gt;
                        <span class="tag">&lt;p&gt;This is a minimal RefEntry.&lt;/p&gt;</span>
                        <span class="tag">&lt;div <span class="att">class="refsection"</span>&gt;
                           <span class="tag">&lt;div <span class="att">class="refsection-titlepage"</span>&gt;
                              <span class="tag">&lt;h4&gt;Sub-subsection&lt;/h4&gt;</span>
                           &lt;/div&gt;</span>
                           <span class="tag">&lt;div <span class="att">class="content"</span>&gt;
                              <span class="tag">&lt;p&gt;This is a minimal RefEntry.&lt;/p&gt;</span>
                           &lt;/div&gt;</span>
                        &lt;/div&gt;</span>
                     &lt;/div&gt;</span>
                  &lt;/div&gt;</span>
               &lt;/div&gt;</span>
            &lt;/div&gt;</span>
            <span class="tag diff">&lt;div <span class="att">class="refsection"</span>&gt;
               <span class="tag diff">&lt;div <span class="att">class="refsection-titlepage"</span>&gt;
                  <span class="tag diff add">&lt;h2&gt;More Description&lt;/h2&gt;</span>
               &lt;/div&gt;</span>
               <span class="tag diff">&lt;div <span class="att">class="<span class="att attchg">content</span>"</span>&gt;
                  <span class="tag diff">&lt;p&gt;<span class="diff txtchg">This is a not-so minimal RefEntry.</span>&lt;/p&gt;</span>
               &lt;/div&gt;</span>
            &lt;/div&gt;</span><span class="diff txtchg">
            </span><span class="tag diff add">&lt;div <span class="att">class="refsection"</span>&gt;
               <span class="tag">&lt;div <span class="att">class="refsection-titlepage"</span>&gt;
                  <span class="tag">&lt;h2&gt;Even More Description&lt;/h2&gt;</span>
               &lt;/div&gt;</span>
               <span class="tag">&lt;div <span class="att">class="content"</span>&gt;
                  <span class="tag">&lt;p&gt;This is an even less minimal RefEntry.&lt;/p&gt;</span>
               &lt;/div&gt;</span>
            &lt;/div&gt;</span>
         &lt;/div&gt;</span>
      &lt;/article&gt;</span>
      <span class="tag nodiff">&lt;script <span class="att">src="../../resources/js/prism.js"</span>/&gt;</span>
   &lt;/body&gt;</span>
&lt;/html&gt;</span></pre></td></tr><tr><td><em>The “current” CSS styling is applied
                  to the this entire document; the results in this
                  column may be anomalous. Also, image links are broken
                  in this column.</em></td><td> </td></tr><tr><td><article id="lessminimal" class="refentry"><div class="refnamediv"><h2>Name</h2><p><span class="refname">Also Minimal</span><span class="refpurpose"><span class="refpurpose-sep"> — </span><span class="refpurpose-text">Another test of a minimal RefEntry</span><span class="refpurpose-punc">.</span></span></p></div><div class="refsection"><div class="refsection-titlepage"><h2>Description</h2></div>

<p>This is a minimal RefEntry.</p>

<div class="refsection"><div class="refsection-titlepage"><h3>Subsection</h3></div>

<p>This is a minimal RefEntry.</p>

<div class="refsection"><div class="refsection-titlepage"><h4>Sub-subsection</h4></div>

<p>This is a minimal RefEntry.</p>
</div>
</div>
</div><div class="refsection"><div class="refsection-titlepage"><h2>More Description</h2></div>

<p>This is a not-so minimal RefEntry.</p>
</div><div class="refsection"><div class="refsection-titlepage"><h2>Even More Description</h2></div>

<p>This is an even less minimal RefEntry.</p>
</div></article><script src="../../resources/js/prism.js"></script></td><td><article id="lessminimal" class="refentry"><div class="content"><div class="refnamediv"><h2>Name</h2><p><span class="refname">Also Minimal</span><span class="refpurpose"><span class="refpurpose-sep"> — </span><span class="refpurpose-text">Another test of a minimal RefEntry</span><span class="refpurpose-punc">.</span></span></p></div><div class="refsection"><div class="refsection-titlepage"><h2>Description</h2></div><div class="content">

<p>This is a minimal RefEntry.</p>

<div class="refsection"><div class="refsection-titlepage"><h3>Subsection</h3></div><div class="content">

<p>This is a minimal RefEntry.</p>

<div class="refsection"><div class="refsection-titlepage"><h4>Sub-subsection</h4></div><div class="content">

<p>This is a minimal RefEntry.</p>
</div></div>
</div></div>
</div></div><div class="refsection"><div class="refsection-titlepage"><h2>More Description</h2></div><div class="content">

<p>This is a not-so minimal RefEntry.</p>
</div></div><div class="refsection"><div class="refsection-titlepage"><h2>Even More Description</h2></div><div class="content">

<p>This is an even less minimal RefEntry.</p>
</div></div></div></article><script src="../../resources/js/prism.js"></script></td></tr></tbody></table><h2>XML source</h2><pre data-src="../src/refentry.005.xml"></pre></body><script type="text/javascript" src="http://code.jquery.com/jquery-1.6.4.min.js"></script><script type="text/javascript" src="http://ajax.googleapis.com/ajax/libs/jqueryui/1.11.4/jquery-ui.min.js"></script><script type="text/javascript" src="../../resources/js/annotation.js"></script><script type="text/javascript" src="../../resources/js/prism.js"></script></html>