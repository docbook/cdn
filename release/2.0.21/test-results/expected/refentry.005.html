<html xmlns="http://www.w3.org/1999/xhtml"><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8" /><title>Also Minimal Title</title><meta charset="utf-8" /><meta name="generator" content="DocBook XSL 2.0 Stylesheets V2.0.17" /><link rel="stylesheet" type="text/css" href="../../resources/css/default.css" /><script type="text/javascript" src="../../resources/js/dbmodnizr.js"></script><link href="../../resources/css/prism.css" rel="stylesheet" type="text/css" /><link href="../../resources/css/db-prism.css" rel="stylesheet" type="text/css" /></head><body><article id="lessminimal" class="refentry"><div class="refnamediv"><h2>Name</h2><p><span class="refname">Also Minimal</span><span class="refpurpose"><span class="refpurpose-sep"> — </span><span class="refpurpose-text">Another test of a minimal RefEntry</span><span class="refpurpose-punc">.</span></span></p></div><div class="refsection"><div class="refsection-titlepage"><h2>Description</h2></div>

<p>This is a minimal RefEntry.</p>

<div class="refsection"><div class="refsection-titlepage"><h3>Subsection</h3></div>

<p>This is a minimal RefEntry.</p>

<div class="refsection"><div class="refsection-titlepage"><h4>Sub-subsection</h4></div>

<p>This is a minimal RefEntry.</p>
</div>
</div>
</div><div class="refsection"><div class="refsection-titlepage"><h2>More Description</h2></div>

<p>This is a not-so minimal RefEntry.</p>
</div><div class="refsection"><div class="refsection-titlepage"><h2>Even More Description</h2></div>

<p>This is an even less minimal RefEntry.</p>
</div></article><script src="../../resources/js/prism.js"></script></body></html>